### PR TITLE
feat: REU student blog posts

### PR DESCRIPTION
**Pull request by Sean1572**

### CODE REVIEW

**Comment by ntlhui** [comment]
When able, please update

File: _layouts/profile-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+{{page.content}}
+
+<h2> Student Researchers </h2>
+
+{% for student in page.students %}
+<div class="floatclearfix">
+    {% if student.photo %}
+    <p style="display: flex; justify-content: center;"><a href="{{ student.photo | absolute_url }}"><img alt="{{student.name}}" class="{{student.align}}" src="{{student.photo | resize: '180x240'}}"></a></p>{% endif %}
```
Comment (ntlhui) [inline _layouts/profile-blog-post.md:19]: Is there a reason we are not converting to .webp?

File: _layouts/profile-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <p>{{student.bio}}</p>
+</div>
+{% endfor %}
+
+{% if page.mentors %}
+    <h2> Mentors </h2>
+    {% for student in page.mentors %}
+    <div class="floatclearfix">
+        {% if student.photo %}
+        <p style="display: flex; justify-content: center;"><a href="{{ student.photo | absolute_url }}"><img alt="{{student.name}}" class="{{student.align}}" src="{{student.photo | resize: '180x240'}}"></a></p>{% endif %}
```
Comment (ntlhui) [inline _layouts/profile-blog-post.md:29]: Is there a reason we are not converting to .webp?

**Review by ntlhui — APPROVED** [review]

File: _layouts/profile-blog-post.md (hunk 2 above)
Comment (Sean1572) [reply _layouts/profile-blog-post.md:29]: No, I'll make an issue to include file conversion for a later pr

**Pull request merged**